index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>KMarketer</title>
     <link rel="stylesheet" href="./assets/styles/styles.css" />
```

Comment:
Since you converted your css into scss you don't need to link the css file anymore